rosetta/services/constants.go

```diff
@@ -6,14 +6,19 @@ const (
 	RosettaVersion = "1.4.5"
 	NodeVersion    = "1.1.0"
 
-	// OpStatusOK is the operation status for successful operations.
-	OpStatusSuccess = "Success"
-	// OpStatusFailed is the operation status for failed operations.
-	OpStatusFailed = "Failed"
-
 	opTransfer = "Transfer"
 	opFee      = "Fee"
 	opReward   = "Reward"
 	opScResult = "SmartContractResult"
 	opInvalid  = "Invalid"
 )
+
+var (
+	opsStatusSuccess = "Success"
+	opStatusFailed   = "Failed"
+
+	// OpStatusOK is the operation status for successful operations.
+	OpStatusSuccess = &opsStatusSuccess
```

Comment:
1. wrong comment
2. why do you pass the address of the `opsStatusSuccess`? In `network_service` you then get the value from the address. I think you can use it directly, without pointers

Reply:
refactored this part of code.